backend-project/small_eod/generic/tests.py

```diff
@@ -22,6 +22,7 @@ def setUp(self):
         self.obj = self.factory_class()
         self.user = getattr(self, "user", UserFactory(username="john"))
         self.client.login(username="john", password="pass")
+        self.response_results_key = "results"
```

Comment:
Ja proponuje zdefiniować to jako atrybut klasy, skoro ma to stałą, niezmienną wartość. Zobacz na https://www.toptal.com/python/python-class-attributes-an-overly-thorough-guide .